gremlin-util/src/test/java/org/apache/tinkerpop/gremlin/structure/io/AbstractTypedCompatibilityTest.java

```diff
@@ -533,7 +533,8 @@ public void shouldReadWriteEmptyList() throws Exception {
         assertEquals(resource, recycled);
     }
 
-    @Test
+    // todo: map contains removed date type, revisit after datetime is properly implemented
```

Comment:
Datetime compatibility tests are enabled along with Element `label` changes in #2780